app/components/tabs_component.rb

```diff
@@ -0,0 +1,17 @@
+class TabsComponent < GovukComponent::Base
+  def initialize(classes: [], html_attributes: {})
```

Comment:
If you like you can ditch this initializer and inherit from `ApplicationComponent`.